product_docs/docs/pem/8/pem_architecture.mdx

```diff
@@ -44,10 +44,15 @@ The PEM server consists of an instance of Postgres, an instance of the Apache we
 
 The instance of Postgres (a database server) and an instance of the Apache web-server ( HTTPD) can be on the same host or on separate hosts.
 
--   **Postgres Instance (Database server)** - This is the backend database server. It hosts a database named **pem** which acts as the repository for PEM Server. The **pem** database contains several schemas that store metric data collected from each monitored host, server, and database.
+-   **Postgres Instance (Database server)** - This is the backend database server. It hosts a database named **pem** which acts as the repository for PEM Server. The **pem** database contains several schemas that store metric data collected from each monitored host, server, and database. 
+    
+    !!! Note
+        All PEM features are available for both backend database servers, EDB Postgres Advanced Server and PostgreSQL. 
```

Comment:
In this instance I think we want to stick closely to the recommended text to avoid inadvertently changing the meaning.
```suggestion
       All the PEM features are available irrespective of which backend database server you select, PostgreSQL or EDB Postgres Advanced Server.
```